.github/workflows/auto-assign-project.yml

```
@@ -1,19 +1,34 @@
+        run: |
+          echo "🔎 Checking PR title: ${{ github.event.pull_request.title }}"
+          TITLE="${{ github.event.pull_request.title }}"
+          if [[ ! "$TITLE" =~ ^(feat|fix|chore|docs|refactor|test|style):\ .+ ]]; then
+            echo "❌ Invalid PR title convention: '$TITLE'"
+            echo "✅ Expected format: 'feat: something here'"
+            exit 1
+          fi
+          echo "✅ PR title is valid."
+
+      - name: 🔍 Check branch name
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)